packages/mobile/e2e/src/Pin.spec.js

```diff
@@ -0,0 +1,24 @@
+import PINChange from './usecases/PINChange'
+import PINRequire from './usecases/PINRequire'
+import { launchApp } from './utils/retries'
+import { quickOnboarding } from './utils/utils'
+
+describe('Given PIN', () => {
+  beforeEach(async () => {
+    await device.uninstallApp()
+    await device.installApp()
+    await launchApp({
+      newInstance: false,
+      permissions: { notifications: 'YES', contacts: 'YES' },
+    })
+    await device.setURLBlacklist(['.*blockchain-api-dot-celo-mobile-alfajores.appspot.com.*'])
```

Comment:
nit: looks like we're duplicating this in a few places now, could be good to have a shared helper.

Reply:
Added as a shared helper in `packages/mobile/e2e/src/utils/utils.js` named `setUrlDenyList`